pkg/build/build.go

```diff
@@ -88,7 +95,7 @@ func (b *Builder) prepareBuildDir() error {
 
 	err := os.MkdirAll(b.combustionDir, os.ModePerm)
 	if err != nil {
-		return fmt.Errorf("creating the build directory structure: %w", err)
+		return fmt.Errorf("creating the build directory structure for combustion: %w", err)
```

Comment:
Nit: Do we need this comment change?

Reply:
not anymore, it was originally changed because I was creating a second dir so I wanted to differentiate them, but I just removed it the second dir creation so I'll change it back